packages/react-components/react-virtualizer/library/src/components/Virtualizer/Virtualizer.types.ts

```diff
@@ -171,6 +189,12 @@ export type VirtualizerConfigProps = {
    * Imperative ref contains our scrollTo index functionality for user control.
    */
   imperativeVirtualizerRef?: RefObject<VirtualizerDataRef>;
+
+  /**
+   * A ref that provides the size of container (vertical - height, horizontal - width), set by a resize observer.
+   * Virtualizer Measure hooks provide a suitable reference.
+   */
+  containerSizeRef: MutableRefObject<number>;
```

Comment:
You're introducing a required property in the `Virtualizer` props types, this is considered a breaking change @Mitch-At-Work , please mark this PR as such, since it'll help us know what PR's are causing breaking changes.

I'm not sure I'm following 100% why this has to be required and why you only accept a ref to a number and not also a number itself. Let's discuss more this modifications 😁